packages/components/src/components/StatusIcon/StatusIcon.jsx

```diff
@@ -12,14 +12,14 @@ limitations under the License.
 */
 
 import {
-  CheckmarkFilled20 as CheckmarkFilled,
-  CheckmarkFilledWarning20 as CheckmarkFilledWarning,
-  CheckmarkOutline20 as CheckmarkOutline,
-  CloseFilled20 as CloseFilled,
-  CloseOutline20 as CloseOutline,
-  Time20 as Pending,
-  WarningAltFilled20 as WarningFilled
-} from '@carbon/icons-react';
+  CheckmarkFilled,
+  CheckmarkFilledWarning,
+  CheckmarkOutline,
+  CloseFilled,
+  CloseOutline,
+  Time as Pending,
+  WarningAltFilled as WarningFilled
+} from '@carbon/react/icons';
```

Comment:
16 is the default size for all these icons now?

Reply:
Good catch, not sure why the code mods didn't handle this.
Fixed in https://github.com/tektoncd/dashboard/pull/3501/commits/36f0edd6fb6971ee4b3cc4ab6ccf5023379acf7e